model/ravelWrap.cc

```diff
@@ -462,7 +462,10 @@ namespace minsky
           {
             vector<size_t> outHandles(dims.size());
             ravel_outputHandleIds(ravel, &outHandles[0]);
-            size_t prevNumElem=v.numElements();
+            size_t prevNumElem;
```

Comment:
Once we get the VariableValue interface right, much of this code can be simplified such as to not care whether the vv is dense or sparse.

Reply:
I've done so through the common dataSize() member that substitutes index.size() and numDenseElements() everywhere.